dataservice/api/diagnosis/models.py

```diff
@@ -0,0 +1,26 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base
+
+
+class Diagnosis(db.Model, Base):
+    """
+    Diagnosis entity.
+
+    :param _id: Unique id assigned by RDBMS
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param created_at: Time of object creation
+    :param modified_at: Last time of object modification
+    :param external_id: Name given to diagnosis by contributor
+    :param primary_diagnosis: Diagnosis of the participant
+    :param progression_or_recurence: yes/no/unknown indicator to identify
+     whether a patient has had a new event after initial treatment.
+    :param days_to_last_followup: Time interval from the date of last follow up
+     to the date of initial pathologic diagnosis, represented as a calculated
+     number of days.
+    """
+
+    __tablename__ = 'diagnosis'
+    external_id = db.Column(db.String(32))
+    primary_diagnosis = db.Column(db.Text)
```

Comment:
This is what we had previously, but I was thinking today maybe we make this several fields:
- `raw_diagnosis_text` for what comes in from input files
- `ICD-10` what we map the raw diagnosis text to this standard
- `MONDO` what we map the raw diagnosis text to this standard

Don't think this should hold up this PR, and the general structure between the raw and the mapped data needs more thought. But noticed as the general setup here is to be many to one with participant and `primary_diagnosis` typically indicates a one to one relationship. So if nothing else should drop the `primary` part if many to one.

Reply:
👍 
And is it correct to model Participant-Diagnosis as a many-to-many relationship?

Reply:
🤔 well I think for the MVP we could just leave it at `primary_diagnosis` and have it be one to one and overall that would be simpler. But that might not set us up right because we know shortly forthcoming data sets will have multiple diagnoses.

Reply:
Hmm just based on the kids first data I think we should leave it as many to many and I can rename `primary_diagnosis` to `diagnosis`. I think this will set us up well for the future and its a very minor change

Reply:
Woops will change Text to Text()